# Define the goal
float32 desired_width
float32 desired_speed
---
# Define the result
bool success
---
